pkg/istio-agent/xds-proxy.go

```diff
@@ -166,12 +168,20 @@ func (p *XdsProxy) StreamAggregatedResources(downstream discovery.AggregatedDisc
 		select {
 		case err := <-upstreamError:
 			// error from upstream Istiod.
-			proxyLog.Infof("Upstream terminated with status %v", err)
+			if isExpectedGRPCError(err) {
```

Comment:
For upstream error, not sure if this is right `"client disconnected"`.

Reply:
I added "transport is closing" to `isExpectedGRPCError` method